packages/web/src/scss/components/Item/index.html

```diff
@@ -1,5 +1,120 @@
 {{#> layout/plain }}
 
+<section class="docs-Section">
+  <h2 class="docs-Heading">Default</h2>
+  <div class="docs-Stack docs-Stack--stretch">
```

Comment:
```suggestion
  <div class="docs-Stack docs-Stack--start">
```

?

But then we may need to also apply `form-fields-theme.$box-field-input-width` (and `max-width: 100%`) on `Item` root.

Reply:
I wouldn't do that. It should be sized as the content imho.